streamdeck_ui/main.ui

```diff
@@ -282,87 +282,89 @@
                </item>
               </layout>
              </item>
-             <item row="1" column="0">
+             <item row="2" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Label:</string>
                </property>
               </widget>
              </item>
-             <item row="1" column="1">
+             <item row="2" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QTextEdit" name="text"/>
                </item>
-               <layout class="QVBoxLayout" name="verticalLayout_4">
-                <item>
+               <item>
```

Comment:
the fix I mentioned in the description of this PR is this. the reason was need without this editors like Qt Creator throw an error saying an unexpected layout found. and not be able to edit the file